boards/pyboard/board.c

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2019 Inria
```

Comment:
Inria has no copyright on this file.

Reply:
actually Kaspar is doing this work paid by Inria + FU collaboration so should the copyright not be instead Inria and FU ?

Reply:
ok, make sense.